src/SSHDebugPS/SSH/SSHConnection.cs

```diff
@@ -46,7 +46,7 @@ public override List<Process> ListProcesses()
             var usernameCommand = _remoteSystem.Shell.ExecuteCommand("id -u -n");
             if (usernameCommand.ExitCode == 0)
             {
-                username = usernameCommand.Output;
+                username = usernameCommand.Output.TrimEnd('\n'); // trim line endings because 'id' command ends with a newline
```

Comment:
Should we trim `\r` also?

Reply:
I can add it. Will we have '\r' on Linux?